workshops/models.py

```diff
@@ -317,6 +317,13 @@ class Event(models.Model):
     latitude = models.FloatField(null=True, blank=True)
     longitude = models.FloatField(null=True, blank=True)
 
+    REPO_REGEX = re.compile(r'https://github\.com/(?P<name>[^/]+)/'
+                            r'(?P<repo>[^/]+)/?')
+    REPO_SCHEMA = 'https://github.com/{name}/{repo}/'
+    WEBSITE_REGEX = re.compile(r'(http|https)://(?P<name>[^.]+)\.github\.'
+                               r'(io|com)/(?P<repo>[^/]+)/?')
```

Comment:
As above, `https?` for the protocol should work.  Also, if they're giving us `name.github.xyz`, then `xyz` can only be `io` not `com`.